docs/installation/linux.md

```diff
@@ -106,6 +107,14 @@ sudo yum remove powershell
 
 [CentOS 7]: https://www.centos.org/download/
 
+Arch Linux
+==========
+Powershell is available from the Arch Linux User Repository (AUR) as a [release](https://aur.archlinux.org/packages/powershell/) or the [latest development build](https://aur.archlinux.org/packages/powershell-git).
```

Comment:
I _personally_ prefer links like:

```markdown
Powershell is available from the Arch Linux User Repository (AUR) as a [release][arch-release]
or the [latest development build][arch-git].

[arch-release]: https://aur.archlinux.org/packages/powershell/
[arch-git]: https://aur.archlinux.org/packages/powershell-git/
```

It's easier to maintain / read in plaintext.